public/vendor/plugins/sortt/sortt.js

```diff
@@ -0,0 +1,62 @@
+/*
+  Sortt
+
+  @License MIT
+  @Author 6543
+  @Repository https://gitea.com/6543/gitea_sortt
+  @Version 1.2
+
+*/
+
+
+// test if jQuery is available
+if(!window.jQuery) {
+	console.log("sortt.js: ERROR no jQuery found!");
+} else {
+
+  //use JQuery to bind event on each <th> with a "data-sortt" attribute
+  $('th').each(function() {
+    if ($(this)[0].dataset.sortt) $(this).on('click', function(e) {
+	    var data = $(this)[0].dataset.sortt
+		  data = data.split(",");
+		  sortt(data[0],data[1],data[2]);
+    });
+  });
+}
+
+//create global function with main routine
+window.sortt=function(normsort,revsort,isdefault){
+  //sortTable [normsort] (revsort) (isdefault)
+
+  //normsort is needet
+  if (!(normsort)) return false;
+
+  //default values of optinal parameters
+  if (!(revsort)) revsort = "";
+
+  // parse URL
+  /*  script check if url has already a sort=
+        NO: if sort=“” indikates default (with * prefix) set url_sort=
+        YES: set url_sort variable
+  */
+  let url = new URL(window.location);
+  let url_sort = url.searchParams.get("sort");
+  if ((url_sort === null) && isdefault) url_sort = normsort;
+
+  // generate new URL
+  /*  script check if url_sort and sort attribute is same
+        NO: generate URL with sort param
+        YES: check if reverse attribute exist -> YES: generate URL with reverse sort param
+  */
+
+  if (url_sort != normsort) {
+    url.searchParams.delete("sort");
```

Comment:
you remove the parameter and add it again, so, you just update it's value, right?
If so, why can't we use `set` method (https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/set) ?